buildrelease.sh

```diff
@@ -65,9 +65,13 @@ fi
 
 ./build.sh $projects
 
+# Retry integration tests up to 3 times as they can sometimes
+# be a little flakey.
 if [[ "$run_tests" = true ]]
 then
-  ./runintegrationtests.sh $projects
+  ./runintegrationtests.sh $projects || true
```

Comment:
Alternatively...

```sh
./runintegrationtests.sh $projects || ./runintegrationtests.sh --retry || ./runintegrationtests.sh --retry
```

? That would only run it as many times as necessary, right?

Reply:
Yea good call, we skip unneeded calls to the script.